run-tests.sh

```diff
@@ -252,7 +252,7 @@ runTestsIfServiceVersion "@coreMetadataPage" "portal" "2.20.8"
 runTestsIfServiceVersion "@indexing" "portal" "2.26.0" "2020.05"
 runTestsIfServiceVersion "@cleverSafe" "fence" "4.22.4" "2020.09"
 runTestsIfServiceVersion "@requestor" "requestor" "1.5.0" "2022.02"
-runTestsIfServiceVersion "@requestor-new" "requestor" "1.5.1" "2022.05"
+runTestsIfServiceVersion "@requestorNew" "requestor" "1.5.1" "2022.05"
```

Comment:
the new feature did not make the 2022.05 cut 😬

Reply:
Oh! I can change that, but however I'm having trouble with this design. Since this annotaions will only work at a test level but not the scenario level. I was wondering if I can just change to  `runTestsIfServiceVersion "@requestor" "requestor" "1.5.0" "2022.02"` to `runTestsIfServiceVersion "@requestor" "requestor" "1.5.1"`

Reply:
i didn't get it, isn't a scenario the same as a test?

we can't change it that way because we need to include monthly releases, that's what almost everyone runs now. We could change `requestorNew` to something a bit more explicit, but other than that i think this is the way to do it 🤔 Or the requestor tests will be skipped for releases < 2022.06

Reply:
oh, i see what you mean. i thought the tag system worked at the scenario level, are you sure?
if not, either sounds fine to me, but for #2 it looks like this is not up to date: https://github.com/uc-cdis/requestor/blob/master/pyproject.toml#L3

Reply:
I also think, once if we are sure that all the commons moved to `requestor 1.5.1` or `2022.06` we can merge the test suite. Thoughts?